### PR TITLE
TN-2782-ironman-v5-migration-batch10

### PR DESCRIPTION
**Pull request by mcjustin**
https://jira.movember.com/browse/TN-2782

### CODE REVIEW

**Comment by mcjustin** [comment]
one minute, something to clean up w/ protocol inheritance...

**Comment by mcjustin** [comment]
Not going to change anything after all. @pbugni if you could please field my comment today.

**Review by pbugni — APPROVED** [review]
Looks good - thanks for addressing white space.

**Comment by mcjustin** [comment]
@ivan-c  this is ready for your Friday

File: portal/config/eproms/Organization.json [hunk 1]
```diff
@@ -405,6 +405,13 @@
         {
           "timezone": "Australia/Brisbane",
           "url": "http://hl7.org/fhir/StructureDefinition/user-timezone"
+        },
+        {
+          "research_protocols": [
+            {"name": "IRONMAN v3", "retired_as_of": "2020-09-29T14:00:00Z"},
```
[the remark this answers is not in the document]
Comment (mcjustin) [reply portal/config/eproms/Organization.json:411]: @pbugni  let me try to flag you here again...

[the remark this answers is not in the document]
Comment (pbugni) [reply portal/config/eproms/Organization.json:411]: this should function as you'd expect.  we look at each level "up" the tree till we find one or more research protocols for the user's organization.

**Review by ivan-c — APPROVED** [review]

**Pull request merged**